openAPI/quran.com/v4.json

```diff
@@ -2690,7 +2690,7 @@
         {
           "name": "recitation_id",
           "in": "path",
-          "description": "Recitation Id, you can get list of all ayah by ayah recitations using this endpoint #endpoint:HLbauN2sdGitPQPPL",
+          "description": "Recitation ID, you can get list of all ayah by ayah recitations using this endpoint #endpoint:HLbauN2sdGitPQPPL",
```

Comment:
"this endpoint #endpoint:HLbauN2sdGitPQPPL" bit needs to be updated.

Reply:
Confused about this, get back to me on Discord